recipes-go/3mdeb-rtectrl/3mdeb-rtectrl_git.bb

```diff
@@ -5,11 +5,15 @@ HOMEPAGE = "https://github.com/3mdeb/RteCtrl/blob/master/README.md"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${S}/src/${GO_IMPORT}/LICENSE;md5=50335162666472aa33383f35cce3e355"
 
-TAG = "0.5.2"
-SRC_URI = "git://git@github.com/3mdeb/RteCtrl.git;protocol=https;branch=master;tag=${TAG}"
+TAG = "0.5.3-rc2"
```

Comment:
@PLangowski let's remove this change (about rtectrl) from this pull request - we did not test it enough to add to this release.

Reply:
you mean remove this commit (https://github.com/3mdeb/meta-rte/commit/ef8d86a0d0651422aa0a65bdd4aeb1895145433a) from the PR?